src/developer-documentation/relayer/relayer.md

```diff
@@ -0,0 +1,156 @@
+## ETH Light Client & Relayer setup & usage example
+
+For now ETH Light Client is integrated only with Brooklyn.  
```

Comment:
@maksym-nyrka Let's make it more clear that these instructions are for Brooklyn (and to mention it's a dev chain), but the process will be similar to Sydney and mainnet.

As it's easy to get lost between chains. Maybe just to make it a bold header or kinda disclaimer.

Reply:
We just don't mention Brooklyn anywhere, so maybe a good place to make some explanation here.